man1/aur-pkglist.1

```diff
@@ -101,6 +101,11 @@ minutes; setting a lower interval with
 thus has no benefit.
 .RE
 .
+.TP
+.BR \-v ", " \-\-verify
+Verify checksums of the compressed list with
+.BR sha256sum (1).
```

Comment:
It might be worth a small note here that `cksum` is generally prefered these days, by GNU, but it is very new.